src/main/java/org/jabref/gui/actions/IntegrityCheckAction.java

```diff
@@ -5,17 +5,18 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-
 import javax.swing.Action;
 import javax.swing.JButton;
 import javax.swing.JCheckBoxMenuItem;
 import javax.swing.JDialog;
 import javax.swing.JOptionPane;
 import javax.swing.JPopupMenu;
+import javax.swing.JProgressBar;
```

Comment:
You somehwere have a checkstyle error here:
`checkstyleMain[ant:checkstyle] [ERROR] /home/travis/build/JabRef/jabref/src/main/java/org/jabref/gui/actions/IntegrityCheckAction.java:8: 'javax.swing.Action' should be separated from previous imports. [ImportOrder]
`
 FAILED

Reply:
If you use Eclipse, exeute ./gradlew eclipse and it will import the correct settings for the checkstyle.